src/styles/style.css

```diff
@@ -1540,3 +1540,18 @@ label > .toggl-button.outlook {
   margin-right: 13px;
   margin-top: 6px;
 }
+
+/********* TURTLE *********/
+.toggl-button.turtle-toggl-start-btn {
+  margin: 10px 0;
+  filter: grayscale(100%);
+  transition: opacity 900ms ease, filter 900ms ease;
```

Comment:
We'd like to use the native styles for now - the `filter:` is good but please can you remove all `transition` properties?

We plan to make updates across global styles and this may include a `transition` then, so it is a good idea 👍